wiki/BanchoBot/id.md

```diff
@@ -1,29 +1,27 @@
+<!-- Informasi mengenai BanchoBot -->
 # BanchoBot
 
-![BanchoBot's player card](BanchoBot.jpg "BanchoBot's player card")
+![Kartu pemain BanchoBot](BanchoBot.jpg "Kartu pemain BanchoBot")
 
-Hostmu untuk [server obrolan osu!](/wiki/Internet_Relay_Chat).
+BanchoBot adalah sebuah bot (yaitu kumpulan perintah yang disusun untuk merespon secara otomatis) yang diprogram oleh **Echo49**, BanchoBot membantu para pemain dalam obrolan dengan menampilkan informasi tertentu dan memberitahukan pesan terkait game. 
```

Comment:
Really minor stuff here but the comma (,) after Echo49 should be a full stop (.) because the content of both senteces are not connected directly to each other hehe